styles/standard.mapcss

```diff
@@ -93,7 +93,10 @@ way|z9-[railway=razed]
 /*************************************************************/
 /* tunnel label (not render if the track has a track number) */
 /*************************************************************/
-way[tunnel=yes][!"railway:track_ref"]::tunnels
+way|z9-[tunnel=yes][!"railway:track_ref"]::tunnels,
```

Comment:
I think you should add railway=rail/disused/abandoned/... here, otherwise e.g. tram tracks would also match this condition (which they should not do).